database/buildSchema/31_review_status_history.sql

```diff
@@ -35,3 +35,10 @@ RETURNS public.review_status AS $$
 	SELECT "status" FROM review_status_history 
 	WHERE review_id = app.id and is_current = true
 $$ LANGUAGE sql STABLE;
+
+-- Function to expose time_create field on review table in GraphQL
+CREATE FUNCTION public.review_time_created(app public.review)
+RETURNS timestamptz AS $$
+	SELECT time_created FROM review_status_history 
```

Comment:
hmm maybe we should change it to time_updated ? (similar to what is being done with review_responses), but happy to merge as is (and make the changes in future PRs

Reply:
never mind, this is fine since review_status_history should just be created (and never changed)